src/docs.md

```diff
@@ -355,6 +355,16 @@ button.textContent = 'Download Excel';
 button.onclick = () => cadenzaClient.downloadData('{embeddingTargetId}', 'excel');
 ```
 
+### Reload the iframe that is used to show Cadenza
+
+<small>API: [CadenzaClient#reload](./classes/CadenzaClient.html#reload)</small>
+
+Reload the iframe that is used to show Cadenza. This can be used to reload the worksheet and it's views.
```

Comment:
it's => its

Also I don't think it's helpful to repeat the heading here.

Reply:
Changed.